app/services/scrapers/low_priority.rb

```diff
@@ -19,7 +19,7 @@ def perform
         county = c.county
 
         case_number = c.case_number
-        court_case = ::CourtCase.find_by!(county_id: county.id, case_number: case_number)
+        court_case = ::CourtCase.find_by!(county_id: c.county_id, case_number: c.case_number)
```

Comment:
what is this?

Reply:
referencing the specific county of the case but I have a variable for that